integration/src/test/scala/ammonite/integration/TestUtils.scala

```diff
@@ -2,11 +2,16 @@ package ammonite.integration
 
 import ammonite.ops._
 import ImplicitWd._
+import ammonite.util.Util
 
 /**
   * Created by haoyi on 6/4/16.
   */
 object TestUtils {
+  val newLine = Util.windowsPlatform match{
```

Comment:
As far as I can tell this is currently unnecessary; it's only used in one place and you surrounded that with an `if (Util.windowsPlatform)` anyway